charts/zitadel/templates/service_login.yaml

```diff
@@ -0,0 +1,33 @@
+{{ if .Values.login.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "zitadel.login.fullname" . }}
+  {{- if .Values.login.service }}
+  {{- with .Values.login.service.annotations }}
+  annotations:
+      {{- toYaml . | nindent 4 }}
+  {{- end }}
+  {{- end }}
+  labels:
+    {{- include "login.labels" . | nindent 4 }}
+    {{- with .Values.login.service.labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+spec:
+  type: {{ .Values.login.service.type }}
+  {{- if and .Values.service.clusterIP (eq .Values.login.service.type "ClusterIP") }}
+  clusterIP: {{ .Values.login.service.clusterIP }}
+  {{- end }}
+  {{- if and .Values.service.externalTrafficPolicy (eq .Values.login.service.type "LoadBalancer") }}
+  externalTrafficPolicy: {{ .Values.login.service.externalTrafficPolicy }}
+  {{- end }}
+  ports:
+    - port: {{ .Values.login.service.port }}
+      targetPort: 3000
```

Comment:
As mentioned in another comment, configurable?

Reply:
Why would you want to configure this?